DFReader.py

```diff
@@ -497,24 +514,23 @@ def check_condition(self, condition):
     def param(self, name, default=None):
         '''convenient function for returning an arbitrary MAVLink
            parameter with a default'''
-        if not name in self.params:
+        if name not in self.params:
             return default
         return self.params[name]
 
+
 class DFReader_binary(DFReader):
     '''parse a binary dataflash file'''
     def __init__(self, filename, zero_time_base=False):
         DFReader.__init__(self)
         # read the whole file into memory for simplicity
-        f = open(filename, mode='rb')
-        self.data = f.read()
+        with open(filename, mode='rb') as f:
```

Comment:
http://stackoverflow.com/questions/7395542/is-explicitly-closing-files-important